Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Fossa CLI Changelog
 
+## Unreleased
```

Comment:
Let's do a release. 

Prefer wording to follow: `$SCOPE:? $DESCRIPTION_AS_A_SENTENCE. ($PR)` 
Refer to: https://github.com/fossas/spectrometer/pull/314#issue-958753432

Reply:
Please have a look but I think I've addressed this.

Reply:
lgtm from me on this. Will defer to Wes for the rest of pr.